### DISCUSSION TITLE
Add external plugins to wheel package openvino-runtime

### DISCUSSION THREAD

**Discussion by mikhailk62** [openvinotoolkit/openvino#14405] [Ideas]
We found a [limitation](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/wheel/setup.py#L53) on adding external plugins (for example we cannot add template and arm plugins) to the python wheel openvino-runtime extension. OpenVINO has only a limited set of internal plugins that are added to the wheel package. And this set cannot be expanded or somehow changed. Also, any user can try to create his own external plugin and may face this problem of adding libraries to the openvino-runtime package. Could you think about adding such functionality?

**Comment by ilya-lavrenov** [comment]
ARM plugin is currently added to wheel package, because it has name CPU and when we build for ARM, ARM CPU plugin is actually installed. I suppose RISC-V plugin will also be named as CPU and transparently installed to wheel package.

As more scalable option - wheel's setup.py can parse plugins.xml file and install all plugins listed there instead of having hardcoded map of plugins. If you interested in, you can start with this solution.
